### DISCUSSION TITLE
Difference to Valetudo

### DISCUSSION THREAD

**Discussion by betterworld2** [rand256/valetudo#416] [General]
Hi,
there has been a lot of development for Valetudo Re as well as Valetudo. 
Can you tell me whether there are still differences or whether it's basically the same with a different name? What are the differences?

**Comment by Belgadon** [comment]
It was Valetudo PLUS other features at the beginning.
The question is: Are features, that are new to Valetudo also implement in Valetudo RE?
Is it still/will it be Valetudo + extra features?
Or are they two completely different projects now now with different features each?

**Reply by rand256** [reply]
> The question is: Are features, that are new to Valetudo also implement in Valetudo RE?

Nope.

**Reply by Belgadon** [reply]
Ok, thanks for clearing things up for me.

**Reply by ottelo9** [reply]
Use Valetudo RE! In the past I used Valetudo (Non RE) for few month. I missed some features that I saw on the Valetudo RE page so I went to RE and I must say, its the better Valetudo !!

E.g. you can use room cleaning and connect Home Assistant (with Alexa) via MQTT (ok you can do this with nonRE, too). But I love it to say: Alexa clean the Dining Room .. ok. And then right after I could say Alexa clean the kitchen. Roborock now cleans the Dining Room and then the kitchen, its wonderful!

**Comment by ch4r13** [comment]
I'am using Valetudo RE, because I have Xiaomi GEN1 and there is a lot more features than on Valetudo... especially remote control, and map loading ... :) and few other things. (voice pack upload, etc...)

**Comment by ottelo9** [comment]
I see that the original Valetudo gots lot of updates ... Anyone try the new update?
